tests/gis_tests/geoadmin/tests.py

```diff
@@ -71,3 +72,14 @@ def test_olwidget_has_changed(self):
         self.assertFalse(has_changed(initial, data_same))
         self.assertFalse(has_changed(initial, data_almost_same))
         self.assertTrue(has_changed(initial, data_changed))
+
+    def test_olwidget_empty_string(self):
+        geoadmin = site._registry[City]
+        form = geoadmin.get_changelist_form(None)({'point': ''})
+        with patch_logger('django.contrib.gis', 'error') as logger_calls:
+            self.assertInHTML(
```

Comment:
I think I'd put `output = str(form['point'])` in the context manager and leave the assertions outside, just for maximum clarity.